vector/src/main/java/im/vector/adapters/RoomInvitationViewHolder.java

```diff
@@ -43,6 +52,16 @@ void populateViews(final Context context, final MXSession session, final Room ro
                        final AbsAdapter.RoomInvitationListener invitationListener, final AbsAdapter.MoreRoomActionListener moreRoomActionListener) {
         super.populateViews(context, session, room, room.isDirectChatInvitation(), true, moreRoomActionListener);
 
+        if (null != room && room.isDirect()) {
+            vRoomAvatar.setVisibility(View.VISIBLE);
+            vRoomAvatarHexagon.setVisibility(View.GONE);
+            VectorUtils.loadRoomAvatar(context, session, vRoomAvatar, room);
```

Comment:
I'm not sure this is relevant to load avatar here, this action is already done by super.populateViews.

Don't you think we should move this code related to the avatar handling in super.populateViews. We already provide the isDirect boolean. We should just add some check on vRoomAvatarHexagon existence before using it

Reply:
The loading of the avatar for the invitations must remain in the view holder dedicated to the display of the invitations. The RoomViewHolder is overridden for invitations, but the specific graphic elements of the invitations have to be placed in the RoomInvitationViewHolder class and not in the RoomViewHolder. 

Idem for the loading of avatar for invitations. When I try to move the loading method for vRoomAvatarHexagon in the populateViews in the RoomViewHolder class, the loading isn't processed correctly and we display a white space instead of the avatar image.